nft_ingester/src/program_transformers/token_metadata/v1_asset.rs

```diff
@@ -367,8 +367,8 @@ pub async fn save_v1_asset<T: ConnectionTrait + TransactionTrait>(
                     creator: Set(c.address.to_bytes().to_vec()),
                     share: Set(c.share as i32),
                     verified: Set(c.verified),
-                    seq: Set(0), // do we need this here @micheal-danenberg?
-                    slot_updated: Set(slot_i),
+                    seq: Set(Some(0)), // do we need this here @micheal-danenberg?
```

Comment:
I feel like you should answer this now 😂

Reply:
We no longer need this as its no longer `NOT NULL` for `asset` and `asset_creators` tables, and it is not used for token-metadata based assets.  So I removed setting it for those tables.